src/plugins/irc-events/topic.js

```diff
@@ -9,11 +9,9 @@ module.exports = function(irc, network) {
 			return;
 		}
 		var from = data.nick || chan.name;
-		var self = false;
-		if (from.toLowerCase() == irc.me.toLowerCase()) {
-			self = true;
-		}
+		var self = (from.toLowerCase() == irc.me.toLowerCase());
```

Comment:
Oh, it's fine :). Sometimes it's easy to make a mistake when interpreting the tone. But yeah, I'll do the changes later.